plugins/versionpress/src/Cli/SilentUpgraderSkin.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace VersionPress\Cli;
+
+class SilentUpgraderSkin extends \WP_Upgrader_Skin {
```

Comment:
Please add simple note what made you use this silent skin here or in `vp:update()` method.